api/app/tests/c_haines/test_bounding_box.py

```diff
@@ -1,7 +1,14 @@
 """ Test the bounding box logic.
 """
 import os
-import gdal
+try:
```

Comment:
See other question re. from osgeo import gdal being enought? Where is this failing?